projects/vdk-plugins/vdk-structlog/src/vdk/plugin/structlog/structlog_plugin.py

```diff
@@ -77,7 +81,71 @@ def process(self, msg, kwargs):
 """
 
 
+class StructlogConfig:
+    def __init__(self, configuration: Configuration):
+        self._configuration = configuration
+        self._is_cloud = configuration.get_value(STRUCTLOG_CONFIG_PRESET) == "CLOUD"
```

Comment:
preset configuration should have the least priority. 

Users should be able to override it either in config.ini or environment variables. 

Currently, it's the opposite if I understand correctly.